client/my-sites/stats/site.jsx

```diff
@@ -371,6 +369,7 @@ class StatsSite extends Component {
 				<PromoCards
 					isJetpack={ isJetpack }
 					isOdysseyStats={ isOdysseyStats }
+					isSecondaryBanner={ true }
```

Comment:
We're using this `isSecondaryBanner` because of the annual page (/stats/day/annualstats/{site}), correct? 
It took me some time to get that. How do you see if we remove this parameter and add some internal check inside `client/my-sites/stats/promo-cards/index.jsx` to see if we're in the desired page?

With that strategy, we could keep the logic of which banner to show inside of promo cards as it already has all the logic.

Reply:
Sounds good, I'll do it